VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -61,11 +63,25 @@ public void render(BaseViewHolder tag, DiscussionThread discussionThread) {
             }
         }
 
-        final int commentColor = discussionThread.getUnreadCommentCount() == 0 ?
-                R.color.edx_grayscale_neutral_light : R.color.edx_brand_primary_base;
-        holder.discussionPostNumCommentsTextView.setText(Integer.toString(discussionThread.getCommentCount()));
-        holder.discussionPostNumCommentsTextView.setTextColor(getContext().getResources().getColor(commentColor));
-        holder.discussionPostCommentIcon.setIconColor(getContext().getResources().getColor(commentColor));
+        {
+            final String text;
+            final Iconify.IconValue icon;
+            final int iconColor;
+            if (voteCountsEnabled) {
+                text = Integer.toString(discussionThread.getVoteCount());
+                icon = Iconify.IconValue.fa_plus;
+                iconColor = discussionThread.isVoted() ? R.color.edx_brand_primary_base : R.color.edx_grayscale_neutral_light;
+            } else {
+                text = Integer.toString(discussionThread.getCommentCount());
+                icon = Iconify.IconValue.fa_comment;
+                iconColor = discussionThread.getUnreadCommentCount() == 0 ? R.color.edx_grayscale_neutral_light : R.color.edx_brand_primary_base;
+
+            }
+            holder.discussionPostNumCommentsTextView.setText(text);
+            holder.discussionPostCommentIcon.setIcon(icon);
```

Comment:
Should we rename these \*comment\* variables to a more generic name now that they're being used for vote counts as well? And perhaps remove the preexisting comment icon from the layout as well